docs/Server.md

```diff
@@ -345,7 +345,7 @@ fastify.get('/', (request, reply) => {
   console.log(request.raw.ips)
   console.log(request.raw.hostname)
   request.raw.log('Hello')
-  reply.res.log('World')
+  reply.raw.log('World')
```

Comment:
```suggestion
  reply.log.info('World')
```

Reply:
I'll not merge this suggestion as in this example we are showing to the user what they can find in the core object. In other parts of the docs, there is the recommended usage of the logger.

Reply:
I did not know we modify the core objects by attaching the logger. That seems silly to me.

Reply:
I think it was for supporting the middlewares.
Since we are removing them, it might be worth starting this discussion again.
Anyhow it can be disabled with [modifyCoreObjects](https://www.fastify.io/docs/latest/Server/#modifycoreobjects).

I'll open an issue to discuss the matter.